### ISSUE TITLE
Use same filters format as docker-py

### ISSUE THREAD

**Issue by olliemath** [aio-libs/aiodocker#276]
..or document the differences between aiodocker's filters and the official python SDK.

## The issue

I would expect aiodocker to have the same behaviour as the standard docker client - namely:
```python
> client = docker.DockerClient()
> client.containers.list(filters={"label": "service=database"})                                                                                                                                              
[<Container: 43b8c03931>]
```

However, I get the following error when trying to filter containers:
```python
> client = aiodocker.Docker()
> await client.containers.list(filter={"label": "service=database"})
DockerError: DockerError(500, "invalid character '\\'' looking for beginning of object key string")
```

## Solutions

This took me a long time to figure out, but looking under the hood, the docker library manually kludges the filters kwarg for the user with this utility function: https://github.com/docker/docker-py/blob/master/docker/utils/utils.py#L382 

Once you know this, you can work around the different API via `filters=json.dumps({"label": ["service=database"]})`, but I feel like users shouldn't have to look at the source code of both libs to figure this difference out. The solution would be either to update to docs, or reproduce the kludge for users coming from the official docker client.

## Environment

Python3.6.7, aiodocker 0.14.0, Docker 18.06.1-ce running on a Linux machine

**Comment by yannickperrenet** [comment]
@olliemath I actually ran into the same issue, but completely missed you mentioning it. 

I opened a PR to fix the issue: https://github.com/aio-libs/aiodocker/pull/543

**Comment by asvetlov** [comment]
Fixed by #543
